src/StreamFactoryInterface.php

```diff
@@ -9,15 +9,11 @@
     /**
      * Create a new stream from a resource or a string.
      *
-     * If the argument is a resource, it MUST be readable and SHOULD be seekable. It MAY be writable.
-     * If the argument is a string, a temporary resource will be created that is writable and seekable.
-     * 
-     * If the arugment is a string it will be interpided as the content of the stream. File names or 
-     * file paths are not supported. 
+     * String content will be used as the body of the stream. Files paths are not supported.
```

Comment:
If we're going to drag mocking into this...

Testing the outcome of a temporary resource being passed into a factory is much easier than testing possible permutations of `$mode`. No matter how you argue it, passing in an existing `resource` is easier to test and (imo) use.

Still waiting for @weierophinney and @opengeek to provide input.